praw/models/reddit/subreddit.py

```diff
@@ -733,9 +733,19 @@ def distinguish(self, thing, how='yes'):
             moderator level distinguish. 'no' removes any distinction. 'admin'
             and 'special' require special user priviliges to use.
 
+        :param sticky: Comment is stickied if True, placing it at the top of
+            the comment page regardless of score. If thing is not a top-level
+            comment, this parameter is silently ignored.
         """
+        data = {'how': how, 'id': thing.fullname}
+        if sticky:
+            thing_type = thing.fullname.split('_', 1)[0]
+            is_comment = (
+                thing_type == self.subreddit._reddit.config.kinds['comment']
```

Comment:
I just realized that the way this is done in `Comment.is_root` is because a `Comment` only has a reference to the parent's ID, and not to an actual PRAW model. So, it's probably best to use `isinstance` here instead?